rpc/README.md

```diff
@@ -0,0 +1,96 @@
+# RPC
+
+**RPC** is an externally-facing component through which clients can interact with the node. It receives client requests 
+(e.g., to synchronize with the latest state of the chain, or to submit transactions), performs basic validation, 
+and forwards the requests to the appropriate internal components.
+**RPC** is one of components of the [Miden node](..).
+
+## Architecture
+
+The Miden node is still under heavy development and current architecture is subject of change. This topic will be 
+filled later.
+
+## Usage
```

Comment:
If the section is Usage, then I recommend to add 2 sub-sections:

Installing the RPC
Running the RPC